src/lib.rs

```diff
@@ -175,8 +175,9 @@ use ndarray_rand::RandomExt;
 pub use variable::{
     Backward, Cat, Convolve, ConvolveWithGroups, Data, Eval, Forward, Gradient, MatMatMul,
     MatMatMulT, Overwrite, Param, Stack, Var, VarDiff, VecMatMul, VecVecMul,
+    Input
 };
-use variable::{Input, InputBackward};
```

Comment:
`Input` should not be in the public API. I'll fix it before merging on main.

Reply:
How are you supposed to load existing arrays? Maybe I am missing something.

Reply:
You can use [neuronika::from_ndarray](https://docs.rs/neuronika/latest/neuronika/fn.from_ndarray.html).